include/vcpkg/base/messages.h

```diff
@@ -991,6 +997,8 @@ namespace vcpkg
                     "{value} is a localized message name like LocalizedMessageMustNotEndWithNewline",
                     "The message named {value} ends with a newline which should be added by formatting "
                     "rather than by localization.");
+    DECLARE_MESSAGE(MissingExtension, (msg::extension), "", "Missing '{extension}' extension.");
+    DECLARE_MESSAGE(MissingHeader, (msg::package_name), "", "Unable to find '{package_name}' header.");
```

Comment:
This isn't a package name, it's the matching header for the file type in question. The original code said "7z header" because it's looking for the header that delimits `.7z` files.